docs/source/config_syntax.md

```diff
@@ -175,6 +176,33 @@ _Description:_ `_requires_`, `_disabled_`, `_desc_`, and `_mode_` are optional k
   - `"debug"` -- execute with debug prompt and return the return value of ``pdb.runcall(_target_, **kwargs)``,
     see also [`pdb.runcall`](https://docs.python.org/3/library/pdb.html#pdb.runcall).
 
+## Multiple config files
+
+_Description:_ Multiple config files may be specified on the command line.
+The content of those config files is being merged. When same keys are specifiled in more than one config file,
+the value associated with the key is being overridden, in the order config files are specified.
+If the desired behaviour is to merge values from both files, the key in second config file should be prefixed with `+`.
+Both values associated with `+`-prefixed key pair must be of `dict` or `list` type.
+`dict` values will be merged via update(), `list` values - concatenated.
+Here's an example. In this case, "amp" value will be overridden by json2 config, and "imports" list will be merged:
+```json
+{
+    "amp": "$True"
+    "imports": [
+	"$import torch"
+    ],
+}
+```
+
+```json
+{
+    "amp": "$False"
+    "+imports": [
+	"$from monai.networks import trt_compile"
+    ],
+}
+```
+
```

Comment:
It may be good to note here:
```
The value types for the merged contents must match. In the above example, it will throw an error if the value type in `"+imports"` is a list.
```